conf/defaults.ini

```diff
@@ -882,6 +882,9 @@ level = info
 # optional settings to set different levels for specific loggers. Ex filters = sqlstore:debug
 filters =
 
+# overrides the default error message shown to users when a sensitive backend error should be obfuscated
```

Comment:
nit: The way I’m reading the comment in its current form is that we are using user_facing_default_error to replace sensitive backend errors. To me this implies that there is a default error message that does not obfuscate sensitive backend errors. As far as I understand, this is not exactly the case. We aren’t currently providing sensitive backend errors on the frontend. 

I rephrased it slightly: “Set the default error message shown to users. This message is displayed instead of sensitive backend errors which should be obfuscated.” I find this clearer. Wdyt?